<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ssid_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ssid:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ssid_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>set AP password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="client_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>client list:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open hotspot</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>MAC ADDR</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="four_g_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open 4G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="password_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>set no Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_Button">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
